tokenapi/decorators.py

```diff
@@ -27,12 +27,12 @@ def _wrapped_view(request, *args, **kwargs):
                 user, token = auth_string.decode().split(':', 1)
 
         if not (user and token):
-            return HttpResponseForbidden("Must include 'user' and 'token' parameters with request.")
+            return JsonError("Must include 'user' and 'token' parameters with request.")
 
         user = authenticate(pk=user, token=token)
         if user:
             request.user = user
             return view_func(request, *args, **kwargs)
 
-        return HttpResponseForbidden()
+        return JsonResponseUnauthorized("You are unauthorized to view this page.")
```

Comment:
The message here should be something like, "Unable to authenticate with given 'user' and 'token' pair."